docs/developing/information.md

```diff
@@ -18,7 +18,7 @@ octez-client get balance for tz1QCVQinE8iVj1H2fckqx6oiM85CNJSK9Sx
 
 ## The RPC interface
 
-[The RPC interface](/architecture/rpc) provides information about the blockchain that nodes use to communicate with each other.
+The [RPC interface](/architecture/nodes#the-rpc-interface) provides information about the blockchain that nodes use to communicate with each other.
```

Comment:
It not only provides information, you also have PUT methods for injecting operations, etc. Maybe start more general, then focus on providing info:
```suggestion
The [RPC interface](/architecture/nodes#the-rpc-interface) allows  nodes use to cooperate with each other and with their clients. 
In particular, many RPCs provide information about the blockchain.
```